.browserslistrc

```diff
@@ -0,0 +1,2 @@
+> 1%
+last 2 versions
```

Comment:
👍 
I suggest to add **iOS 12+ mobile Safari** rule, since iOS app works on iOS 12+.

Reply:
Done